### ISSUE TITLE
Wire A1 minimal LM Pylon manifests, artifact ids, and signed access

### ISSUE THREAD

**Issue by AtlantisPleb** [OpenAgentsInc/openagents#4457]
## Objective

Make Pylon execute A1 minimal LM assignments through the existing machine
manifest and artifact resolver path.

## Scope

- Extend `PylonTrainingRunManifestV1` or manifest metadata for the A1 minimal
  LM run definition ref and expected artifacts.
- Include tokenizer, dataset, base checkpoint, assignment shard/range, expected
  output artifact kind, and validator target fields.
- Use logical artifact ids and signed-access routes for uploads/downloads.
- Map Psionic local-update and support artifacts to
  `PylonTrainingArtifactKind`/class values.
- Avoid source-host-local path assumptions.

## Likely files

- `crates/openagents-kernel-core/src/pylon_training.rs`
- `apps/pylon/src/lib.rs`
- `apps/pylon/src/training_trn_mapping.rs`
- `apps/nexus-control/src/lib.rs`
- `docs/plans/compute-training-artifact-resolver-contract.md`
- `docs/plans/compute-training-signed-artifact-access-contract.md`

## Acceptance criteria

- A Pylon can materialize an A1 minimal LM assignment from artifact ids alone.
- A local-update worker uploads the expected artifact class through signed
  access.
- A support worker uploads the expected support artifact class through signed
  access.
- Resolver and signed-access tests cover the new artifact kinds or mappings.

## Dependencies

- Psionic local-update receipts issue: https://github.com/OpenAgentsInc/psionic/issues/951
- Psionic verifier-first support artifacts issue: https://github.com/OpenAgentsInc/psionic/issues/954
- OpenAgents work-class mapping issue: https://github.com/OpenAgentsInc/openagents/issues/4455

## Source

Created from `docs/autopilot/2026-04-27-a1-minimal-distributed-lm-issue-backlog.md`, section 12: OpenAgents Pylon Manifest And Artifact Wiring.

**Comment by AtlantisPleb** [comment]
Implemented on main in commit 2f69fc0f1 (Wire A1 manifest artifacts and signed access).

Summary:
- Added A1 minimal LM work-unit metadata to `PylonTrainingRunManifestV1`, including tokenizer/dataset/validation digests, base checkpoint ref, assignment shard/range, input artifact ids, expected output artifact ids, and validator targets.
- Added typed artifact kinds/classes for tokenizer, tokenized dataset shard, validation set, model/optimizer config, and `support_bundle`.
- Wired Pylon terminal artifact upload selection so local-update work writes `local_update` and verifier/support work writes `support_bundle` through Nexus signed access.
- Extended Nexus signed-access route coverage for `support_bundle`.
- Updated artifact resolver, signed-access, run-definition, and weak-device docs for the new A1 artifact contract.

Verification:
- `rustfmt --edition 2024 crates/openagents-kernel-core/src/pylon_training.rs apps/pylon/src/lib.rs apps/nexus-control/src/lib.rs --check`
- `git diff --check`
- `cargo test -p openagents-kernel-core a1_minimal --lib`
- `cargo test -p pylon a1_minimal_artifact_courier_uploads_local_update_and_support_via_signed_access`
- `cargo test -p nexus-control kernel_compute_training_artifact_signed_access_route_issues_google_v4_urls`
- `cargo test -p pylon training_artifact_courier_accepts_explicit_bearer_token_without_adc`
